admin/installer/setup_client.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# Setup up a client compute instance to deploy sample workloads on an install
+# cluster using docker-compose.
+
+set -ex
+
+DOCKER_CERT_HOME="/root/.docker"
+UBUNTU_HOME="/home/ubuntu"
+
+# Get Postgres image.
+apt-get update
+sudo apt-get install -y postgresql-client
+
+# Get docker-compose.
+curl -L https://github.com/docker/compose/releases/download/1.5.2/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
+chmod +x /usr/local/bin/docker-compose
+
+mkdir -p /home/ubuntu/postgres
+curl https://raw.githubusercontent.com/ClusterHQ/flocker/flocker-cloudformation-FLOC-3709/admin/installer/postgres/docker-compose-node0.yml > /home/ubuntu/postgres/docker-compose-node0.yml
```

Comment:
Probably also meant to remove these curl lines? Luckily curl doesn't fail on not founds, but we're just adding random files with the content `Not Found` to the image now:

```
 ♦ ~$ curl https://raw.githubusercontent.com/ClusterHQ/flocker/flocker-cloudformation-FLOC-3709/admin/installer/postgres/docker-compose-node0.yml
Not Found ♦ ~$ echo $?
0
 ♦ ~$ 
```